config/routes.rb

```diff
@@ -1,4 +1,27 @@
 Rails.application.routes.draw do
+
+  root 'questions#index'
+
+  resources :users, only: [:new, :create, :show]
+  resources :sessions, only: [:new, :create, :destroy]
+
+  get 'questions/recent'=> 'questions#recent', as: 'recent_questions'
+
+  resources :questions do
+    resources :answers, except: [:index, :show]
+    resources :comments, except: [:show,:index]
+  end
+
+  get 'answers/flag_favorite' => 'answers#flag_favorite', as: 'flag_favorite'
```

Comment:
It would be preferable to do
```
member do 
  get 'flag_favorite'
end
```

That would generate a route like `/answers/:id/flag_favorite`